flow/energy_models/toyota_energy.py

```diff
@@ -1,50 +1,58 @@
+"""Script containing the Toyota energy classes."""
 import dill as pickle
 import boto3
-import botocore
-import math
-import random
-import statistics
-import numpy as np
-from scipy.interpolate import interp1d
-from collections import namedtuple
 from flow.energy_models.base_energy import BaseEnergyModel
 import os
 from abc import ABCMeta, abstractmethod
 
-class ToyotaModel(BaseEnergyModel):
+
+class ToyotaModel(BaseEnergyModel, metaclass=ABCMeta):
+    """Base Toyota Energy model class."""
 
     def __init__(self, kernel, filename=None):
         self.k = kernel
 
         # download file from s3 bucket
         s3 = boto3.client('s3')
-        s3.download_file('toyota.restricted', filename,'file.pkl') #move to init
-        with open('file.pkl','rb') as file:
-            self.toyota_energy = pickle.load(file) #self.prius_energy
+        s3.download_file('toyota.restricted', filename, 'temp.pkl')
+        with open('temp.pkl', 'rb') as file:
+            self.toyota_energy = pickle.load(file)
+
         # delete pickle file
-        os.remove(file.pkl)
-    
+        os.remove(file)
+
     @abstractmethod
     def get_instantaneous_power(self, accel, speed, grade):
+        """See parent class."""
         pass
 
 
 class PriusEnergy(ToyotaModel):
-    
+    """Toyota Prius (EV) energy model class."""
+
     def __init__(self, kernel, soc=0.9):
-        super(PriusEnergy, self).__init__(kernel, filename = 'prius_test.pkl')
-        self.soc = soc 
+        super(PriusEnergy, self).__init__(kernel, filename='prius_ev.pkl')
+        self.soc = soc
 
     def get_instantaneous_power(self, accel, speed, grade):
+        """See parent class."""
         socdot = self.toyota_energy(self.soc, accel, speed, grade)
         self.soc -= socdot * self.k.env.sim_step
+        # FIXME (Joy): convert socdot to power
```

Comment:
Are we taking care of this while Joy's on vacation?

Reply:
My plan is to leave it unfixed since it isn't really part of the MVP